Source/blargg.json

```diff
@@ -0,0 +1 @@
254-2 CB AB:254-2 CB AC:254-2 CB AD:254-2 CB AF:254-2 CB B0:254-2 CB B1:254-2 CB B2:254-2 CB B3:254-2 CB B4:254-2 CB B5:254-2 CB B7:254-2 CB B8:254-2 CB B9:254-2 CB BA:254-2 CB BB:254-2 CB BC:254-2 CB BD:254-2 CB BF:254-2 CB C0:254-2 CB C1:254-2 CB C2:254-2 CB C3:254-2 CB C4:254-2 CB C5:254-2 CB C7:254-2 CB C8:254-2 CB C9:254-2 CB CA:254-2 CB CB:254-2 CB CC:254-2 CB CD:254-2 CB CF:254-2 CB D0:254-2 CB D1:254-2 CB D2:254-2 CB D3:254-2 CB D4:254-2 CB D5:254-2 CB D7:254-2 CB D8:254-2 CB D9:254-2 CB DA:254-2 CB DB:254-2 CB DC:254-2 CB DD:254-2 CB DF:254-2 CB E0:254-2 CB E1:254-2 CB E2:254-2 CB E3:254-2 CB E4:254-2 CB E5:254-2 CB E7:254-2 CB E8:254-2 CB E9:254-2 CB EA:254-2 CB EB:254-2 CB EC:254-2 CB ED:254-2 CB EF:254-2 CB F0:254-2 CB F1:254-2 CB F2:254-2 CB F3:254-2 CB F4:254-2 CB F5:254-2 CB F7:254-2 CB F8:254-2 CB F9:254-2 CB FA:254-2 CB FB:254-2 CB FC:254-2 CB FD:254-2 CB FF:254-2 \nFailed\n", "tests/BlarggROMs/mem_timing/mem_timing.gb": "mem_timing\n\n01:01  02:01  03:01  \n\nFailed ", "tests/BlarggROMs/mem_timing/individual/02-write_timing.gb": "02-write_timing\n\n36:2-3 E0:2-3 EA:2-4 \nFailed\n", "tests/BlarggROMs/mem_timing/individual/01-read_timing.gb": "01-read_timing\n\nF0:2-3 FA:2-4 CB 46:2-3 CB 4E:2-3 CB 56:2-3 CB 5E:2-3 CB 66:2-3 CB 6E:2-3 CB 76:2-3 CB 7E:2-3 \nFailed\n", "tests/BlarggROMs/mem_timing/individual/03-modify_timing.gb": "03-modify_timing\n\n35:0/0-2/3\n34:0/0-2/3\nCB 06:0/0-3/4\nCB 0E:0/0-3/4\nCB 16:0/0-3/4\nCB 1E:0/0-3/4\nCB 26:0/0-3/4\nCB 2E:0/0-3/4\nCB 36:0/0-3/4\nCB 3E:0/0-3/4\nCB 86:0/0-3/4\nCB 8E:0/0-3/4\nCB 96:0/0-3/4\nCB 9E:0/0-3/4\nCB A6:0/0-3/4\nCB AE:0/0-3/4\nCB B6:0/0-3/4\nCB BE:0/0-3/4\nCB C6:0/0-3/4\nCB CE:0/0-3/4\nCB D6:0/0-3/4\nCB DE:0/0-3/4\nCB E6:0/0-3/4\nCB EE:0/0-3/4\nCB F6:0/0-3/4\nCB FE:0/0-3/4\n\nFailed\n", "tests/BlarggROMs/cpu_instrs/cpu_instrs.gb": "cpu_instrs\n\n01:ok  02:05  03:ok  04:ok  05:ok  06:ok  07:ok  08:ok  09:ok  10:ok  11:ok  \n\nFailed ", "tests/BlarggROMs/cpu_instrs/individual/02-interrupts.gb": "02-interrupts\n\n\nHALT\n\nFailed ", "tests/BlarggROMs/cpu_instrs/individual/07-jr,jp,call,ret,rst.gb": "Passed", "tests/BlarggROMs/cpu_instrs/individual/09-op r,r.gb": "Passed", "tests/BlarggROMs/cpu_instrs/individual/11-op a,(hl).gb": "Passed", "tests/BlarggROMs/cpu_instrs/individual/10-bit ops.gb": "Passed", "tests/BlarggROMs/cpu_instrs/individual/04-op r,imm.gb": "Passed", "tests/BlarggROMs/cpu_instrs/individual/01-special.gb": "Passed", "tests/BlarggROMs/cpu_instrs/individual/06-ld r,r.gb": "Passed", "tests/BlarggROMs/cpu_instrs/individual/03-op sp,hl.gb": "Passed", "tests/BlarggROMs/cpu_instrs/individual/08-misc instrs.gb": "Passed", "tests/BlarggROMs/cpu_instrs/individual/05-op rp.gb": "Passed", "tests/BlarggROMs/mem_timing-2/rom_singles/02-write_timing.gb": "Timeout:\n", "tests/BlarggROMs/mem_timing-2/rom_singles/01-read_timing.gb": "Timeout:\n", "tests/BlarggROMs/mem_timing-2/rom_singles/03-modify_timing.gb": "Timeout:\n", "tests/BlarggROMs/mem_timing-2/mem_timing.gb": "Timeout:\n"}
```

Comment:
This is probably automatically generated and read so there's no need for pretty printing, but it's missing a trailing newline.

Reply:
Not necessary